### PR TITLE
Faster `highlight-collaborators-and-own-conversations`

### PR DESCRIPTION
**Pull request by yakov116**
https://github.com/sindresorhus/refined-github/pull/3505#issuecomment-687302600

### CODE REVIEW

File: source/features/highlight-collaborators-and-own-conversations.tsx [hunk 1]
```diff
@@ -53,6 +49,10 @@ void features.add({
 	include: [
 		pageDetect.isRepoConversationList
 	],
+	exclude: [
+		() => select.exists('.blankslate')
```
Comment (fregante) [inline source/features/highlight-collaborators-and-own-conversations.tsx:53]: This won't be found if we don't await dom ready, so it can be removed

Comment (yakov116) [reply source/features/highlight-collaborators-and-own-conversations.tsx:53]: Tested, It works before dom ready

**Comment by yakov116** [comment]
>Merge branch 'master' into **hilighcolab**

Oh my that branch name 🤦

File: source/features/highlight-collaborators-and-own-conversations.tsx (hunk 1 above)
Comment (fregante) [reply source/features/highlight-collaborators-and-own-conversations.tsx:53]: Race condition warning‼️

If you're ok with it working sometimes and don't come back in 3 days with another PR that's fine. Of course this expects half the page to be loaded, so it's not gonna work 100% of the time

Comment (fregante) [reply source/features/highlight-collaborators-and-own-conversations.tsx:53]: Either way, there's no error.

Best case scenario, it avoids an API call.
Worst case scenario, there's an unnecessary API call.

**Pull request merged**